client-course-schedulizer/src/utilities/helpers/caseFunctions.ts

```diff
@@ -27,12 +27,36 @@ const thursReg = RegExp("[Tt][Hh]|[Rr]");
 const friReg = RegExp("[Ff]");
 const satReg = RegExp("[Ss](?![Uu])");
 
-export const startTimeCallback = (value: string, { firstMeeting }: CaseCallbackParams) => {
-  firstMeeting.startTime = startTimeCase(value);
+const createMeetings = (value: string, { meetings }: CaseCallbackParams): string[] => {
+  const valueParts = value.split("\n");
+  for (let i = 0; i < valueParts.length; i += 1) {
+    if (meetings.length <= i) {
+      // If there aren't enough meetings, create a new one
+      meetings.push({
+        days: [],
+        duration: 0,
+        location: { building: "", roomCapacity: 0, roomNumber: "" },
+        startTime: "",
+      });
+    }
+  }
```

Comment:
I'd suggest making this for loop a forEach since it is similar to what we do elsewhere in the app. If you disagree, that's fine too.

```
  valueParts.forEach((_, i) => {
    if (meetings.length <= i) {
      // If there aren't enough meetings, create a new one
      meetings.push({
        days: [],
        duration: 0,
        location: { building: "", roomCapacity: 0, roomNumber: "" },
        startTime: "",
      });
    }
  });
```

Reply:
2453811